<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlgCustomerRecharge</class>
 <widget class="QDialog" name="dlgCustomerRecharge">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>550</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1024</width>
    <height>550</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1024</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>充值</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QWidget" name="widget_2" native="true">
     <property name="font">
      <font>
       <family>微软雅黑</family>
      </font>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="1" column="1">
       <widget class="QLineEdit" name="LeCardMoney">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>微软雅黑</family>
         </font>
        </property>
       </widget>
      </item>
      <item row="0" column="2" rowspan="4">
       <widget class="QWidget" name="widget_3" native="true">
        <property name="font">
         <font>
          <family>微软雅黑</family>
         </font>
        </property>
        <widget class="QPushButton" name="BtnRecharge">
         <property name="geometry">
          <rect>
           <x>60</x>
           <y>10</y>
           <width>120</width>
           <height>60</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>120</width>
           <height>60</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>充值</string>
         </property>
        </widget>
        <widget class="QPushButton" name="BtnPayForHaveNotPay">
         <property name="geometry">
          <rect>
           <x>220</x>
           <y>10</y>
           <width>120</width>
           <height>60</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>120</width>
           <height>60</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>客户缴费</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton">
         <property name="geometry">
          <rect>
           <x>60</x>
           <y>100</y>
           <width>120</width>
           <height>60</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>120</width>
           <height>60</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>取衣</string>
         </property>
        </widget>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="LeCardID">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>微软雅黑</family>
         </font>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="LeInput">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>微软雅黑</family>
         </font>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="LeRecharge">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>微软雅黑</family>
         </font>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_8">
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>微软雅黑</family>
         </font>
        </property>
        <property name="text">
         <string>卡号</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_9">
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>微软雅黑</family>
         </font>
        </property>
        <property name="text">
         <string>卡金额</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_10">
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>微软雅黑</family>
         </font>
        </property>
        <property name="text">
         <string>充值额</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_11">
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>微软雅黑</family>
         </font>
        </property>
        <property name="text">
         <string>收银额</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QWidget" name="widget" native="true">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>240</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
      </font>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="2">
       <widget class="QLabel" name="label_6">
        <property name="font">
         <font>
          <family>微软雅黑</family>
         </font>
        </property>
        <property name="text">
         <string>姓名</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLineEdit" name="LeSpend">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="font">
         <font>
          <family>微软雅黑</family>
         </font>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="LePhone">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="font">
         <font>
          <family>微软雅黑</family>
         </font>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QLineEdit" name="LeDiscount">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="font">
         <font>
          <family>微软雅黑</family>
         </font>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <family>微软雅黑</family>
         </font>
        </property>
        <property name="text">
         <string>性别</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <family>微软雅黑</family>
         </font>
        </property>
        <property name="text">
         <string>电话号码</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QLineEdit" name="LeCredit">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="font">
         <font>
          <family>微软雅黑</family>
         </font>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="LeGender">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="font">
         <font>
          <family>微软雅黑</family>
         </font>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="LeAddress">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="font">
         <font>
          <family>微软雅黑</family>
         </font>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="label_4">
        <property name="font">
         <font>
          <family>微软雅黑</family>
         </font>
        </property>
        <property name="text">
         <string>客户信用</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <family>微软雅黑</family>
         </font>
        </property>
        <property name="text">
         <string>所住区域</string>
        </property>
       </widget>
      </item>
      <item row="2" column="5">
       <widget class="QPushButton" name="BtnSelectCustomer">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>30</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>微软雅黑</family>
         </font>
        </property>
        <property name="text">
         <string>选择客户</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="label_5">
        <property name="font">
         <font>
          <family>微软雅黑</family>
         </font>
        </property>
        <property name="text">
         <string>消费总额</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLineEdit" name="LeName">
        <property name="font">
         <font>
          <family>微软雅黑</family>
         </font>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QLabel" name="label_7">
        <property name="font">
         <font>
          <family>微软雅黑</family>
         </font>
        </property>
        <property name="text">
         <string>折扣率</string>
        </property>
       </widget>
      </item>
      <item row="1" column="5">
       <widget class="QTextBrowser" name="textBrowser">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>60</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>微软雅黑</family>
         </font>
        </property>
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="html">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'微软雅黑'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;请只通过以下按钮进行选择客户！&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="openLinks">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
